### PR TITLE
twitter cards: changes

### LINKED ISSUE #4259: Changes on the Twitter cards
After review the existing Twitter cards we propose the next changes:

1) When the creator twitter handler is not available leave the creator content empty, that way it will not display cartodb as the creator.

2) On the title do not add the string 'CartoDB · ' to the title, just only the title of the visualization. Right now is doing like the title of the webpage, just use the title.

3) On the description, try to use the description as specified by the user on the metadata, and always attach at the end after the description "Map created by <username> using @CartoDB"

two examples for description, one when there is a description and the other when there is not.

"Very long description bla bla bla. Map created by jatorre using @CartoDB"

"Map created by jatorre using @CartoDB"

Thanks @matallo !

### PR DESCRIPTION
**Pull request by matallo**
close #4259 

CR @jatorre @CartoDB/frontend

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4677/))

File: app/views/admin/visualizations/public_map.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<%= content_for(:title) do %>CartoDB · <%= @visualization.name.gsub(/_/," ") %><% end %>
+<%= content_for(:title) do %><%= @visualization.name.gsub(/_/," ") %><% end %>
```
Comment (javierarce) [inline app/views/admin/visualizations/public_map.html.erb]: I think here we don't need the `gsub` https://github.com/CartoDB/cartodb/blob/master/app/views/admin/visualizations/public_map.html.erb#L38

**Comment by javierarce** [comment]
:+1: code looks fine, I just added one suggestion.

**Comment by matallo** [comment]
suggestions taken into account, thanks @javierarce!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4680/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4681/))

**Pull request merged**